gtk4/src/auto/entry_buffer.rs

```diff
@@ -75,27 +69,6 @@ pub trait EntryBufferExt: 'static {
 }
 
 impl<O: IsA<EntryBuffer>> EntryBufferExt for O {
-    fn emit_deleted_text(&self, position: u32, n_chars: u32) {
```

Comment:
I moved those to manual as well to make use of u16 so that the api is consistent